### DISCUSSION TITLE
Weekly blog posts! - Blog - GrahamSH

### DISCUSSION THREAD

**Discussion by giscus** [GrahamSH-LLK/grahamsh.com#20] [General]
# Weekly blog posts! - Blog - GrahamSH

https://grahamsh.com/blog/weekly_post

**Comment by TheSeagullDev** [comment]
coolio woolio

**Comment by retronbv** [comment]
Why ping :eeeee:

**Comment by Explosion-Scratch** [comment]
Post cool stuff! E.g. cool bash scripts, JavaScript functions, guides for how to use nuxt and vue and stuff, etc

**Reply by Explosion-Scratch** [reply]
ok sorry i self reacted

**Reply by retronbv** [reply]
> ok sorry i self reacted

Lmao

**Reply by Explosion-Scratch** [reply]
=|
